README.md

```diff
@@ -102,7 +67,7 @@ The AWS IoT Device Client currently works by default on IoT devices with common
 To use the AWS IoT Device Client, you'll need to compile an executable using the source code provided by this
```

Comment:
You could compile w/ them static or shared, that's up to the customer.  Regardless of location required during runtime

Reply:
I see, so they are a requirement, in that case I think that part `Note: CI Build uses` is confusing(at least for me), would probably be clearer to just write

### Minimum Requirements

* C++ 11 or higher
* [CMake](https://cmake.org/) 3.10+
* OpenSSL 1.1.1
* [aws-iot-device-sdk-cpp-v2](https://github.com/aws/aws-iot-device-sdk-cpp-v2) commit hash located in `CMakeLists.txt.awssdk`

Reply:
So it's complicated because you could potentially use something else besides Openssl entirely if you so choose.  Similarly there is a potential that different versions of the sdk could  work fine

Reply:
I think (and correct me if I'm wrong) that these requirements are about what we support not what could be done, for example I'm pretty sure there are tools that can convert c++11 code to c++98 compatible code but we still say clearly that c++11 or hight is required, so if we support multiple TLS implementations, that's fine, and we can specify which ones we support building with any commit we can say that, if we support only subset of commits, then let's specify that as well. But we shouldn't leave it vague like that.

Reply:
We don't currently have infrastructure for testing other TLS stacks or versions of the SDK so we don't really know for certain.  Which is why I think it's better to be vague, more of a "If you want to try it, it might work but we don't actively support that in testing or our CI"  

Also for 11->98 I've never heard of anything like that, at least nothing that works well.  Our requirement on c++11 was chosen because the SDK is the same.  Cmake aswell,

Reply:
Then we need to ask ourselves and figure out what implementation do we support, leaving it vague is just plain confusing.

Reply:
I suppose we could change it to match your earlier message.  and then a note at the end that other TLS stacks and versions could work but we don't actively support them

Reply:
Yeah, I think that would make it clear. Thanks!